docs/theming.md

```diff
@@ -23,3 +23,53 @@ To define a theme for Riot:
 
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
+
```

Comment:
can you do me a favour and on lines 16-18 wrap `_teal.scss`, `_base.scss`, and `_dark.scss` in backticks to avoid infinite italics?